docs/environment-variables.md

```diff
@@ -23,6 +23,7 @@ This document outlines environment variables that can be used to customize behav
 | `ARGO_AGENT_CPU_LIMIT`                   | `resource.Quantity` | `100m`                                                                                      | CPU resource limit for the agent.                                                                                                                                                                                                                                        |
 | `ARGO_AGENT_MEMORY_LIMIT`                | `resource.Quantity` | `256m`                                                                                      | Memory resource limit for the agent.                                                                                                                                                                                                                                     |
 | `ARGO_POD_STATUS_CAPTURE_FINALIZER`      | `bool`              | `false`                                                                                     | The finalizer blocks the deletion of pods until the controller captures their status.
+| `ARGO_TEMPLATE_WITH_INPUTS_PARAMETERS`      | `bool`              | `true`                                                                                     | Whether to keep inputs.parameters inside the ARGO_TEMPLATE environment variable of pods.
```

Comment:
```suggestion
| `ARGO_TEMPLATE_WITH_INPUTS_PARAMETERS`      | `bool`              | `true`                                                                                     | Whether to keep `inputs.parameters` inside the `ARGO_TEMPLATE` environment variable of pods
```

These should have backticks, and then the `.spelling` entries are not needed.
That it pops up as a spelling issue can indicate a different issue, like lack of backticks.

Reply:
https://github.com/argoproj/argo-workflows/pull/13761 addresses this